tests/ui/conftest.py

```diff
@@ -100,12 +101,17 @@ def user(
     }
     assert requests.codes.created == response.status_code
     user.update(response.json())
-    print(user)
     return user
 
 
 @pytest.fixture(scope='function')
-def our_live_server(request):
+def live_server(request):
+    """
+        This fixture overrides the live_server fixture provided by
+        pytest_django. live_server allows us to create a running version of the
+        addons django application within pytest for testing.
+    """
+
     import django
     request.getfixturevalue('transactional_db')
```

Comment:
Is `transactional_db` needed here? If so it should be in the fixture arguments rather than using `request.getfixturevalue`. Also the django import looks to be unused?